kubernetes-templates/ghost/0/rancher-compose.yml

```diff
@@ -0,0 +1,29 @@
+.catalog:
+  name: Ghost
+  version: 0.7.8-rancher1
+  description: Open source publishing platform
+  questions:
+    - variable: "mysql_root_pass"
+      label: "MySQL root password"
+      required: true
+      type: "string"
+    - variable: "ghost_db"
+      label: "Ghost database name on mysql"
+      required: true
+      type: "string"
+      default: "ghost"
+    - variable: "ghost_db_user"
+      label: "Ghost database username"
+      required: true
+      type: "string"
+      default: "ghostuser"
+    - variable: "ghost_db_pass"
+      label: "Ghost database password"
+      required: true
+      type: "string"
```

Comment:
Same as above. The `description` field is a good place to document the default password (field descriptions appear below each field in blue)